Sources/Layouts/TextViewLayout.swift

```diff
@@ -130,6 +130,10 @@ open class TextViewLayout<TextView: UITextView>: BaseLayout<TextView>, Configura
     // MARK: - overriden methods
 
     open override func configure(view textView: TextView) {
+        /// `isSelectable` is default to false, but it allows overriding in `configure` block
+        /// By enabling it, it may have side effects
```

Comment:
Can you either document what kind of side effects or just remove `textView.isSelectable = false` entirely.

If it is safe for isSelectable to be true, then we should probably just remove this.